arrayMethod.js

```diff
@@ -0,0 +1,128 @@
+//1
+const plus10 = (arr) => {
+  return arr.map((el) => {
+    return el + 10
+  })
+}
+console.log(plus10([1,2,3]));
+
+//2
+
+const emptyStr = (arr) => {
+  return arr.map((el) =>{
+      if(typeof el === "string" ){
+      return el
+    }else{
+      return ""
+    }
+  })
+  }
+//}
+console.log(emptyStr(["1", "b", true, {name: "cat"}]));
+
+//3
+
+const evenNum = (arr) => {
+  return arr.filter((el) => {
+    return el % 2 === 0;
+  })
+}
+console.log(evenNum([1,2,3,4,5]));
+
+//4
+const sameArray = (arr) => {
+  return arr.every((el) => {
+    if (el === arr[el]){
```

Comment:
This function doesn't really work. I see what you were trying to do though. You can try arr[0]. This is going to work. el is just the element in the array you're looking at. Lets imagine you're comparing strings and this function wouldn't work.